plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Users.js

```diff
@@ -411,7 +411,7 @@ Ext.define('NX.coreui.controller.Users', {
         win = form.up('nx-coreui-user-add');
 
     if (win) {
-      me.loadStoreAndSelect(action.result.data.userId, false);
+      me.loadStoreAndSelect(action.result.data.userId, true);
```

Comment:
any use cases left for the 'navigate' param to be false? If not, can probably ditch that part of the code